lib/matplotlib/contour.py

```diff
@@ -764,23 +767,20 @@ def __init__(self, ax, *args,
                 _api.warn_external('linewidths is ignored by contourf')
             # Lower and upper contour levels.
             lowers, uppers = self._get_lowers_and_uppers()
-            self.set(
-                edgecolor="none",
-                # Default zorder taken from Collection
-                zorder=kwargs.pop("zorder", 1),
-                rasterized=kwargs.pop("rasterized", False),
-            )
+            self.set(edgecolor="none")
+            self.set(**kwargs)  # Let user-set values override defaults.
```

Comment:
This is on both sides of the `if` (though without the comment later), so can move both outside?

Reply:
sure